test/contract-wrapper-tests/contract-functions-tests.js

```diff
@@ -0,0 +1,225 @@
+            let expectedResult = `Hello ${parameter}`;
+
+            let result = await deployedContract.sayHello(parameter);
+		    assert.equal(result, expectedResult, "Result is not expected one.");
+        });
+
+        it("Should execute function that accept 'int/ints' as parameter successfully", async () => {
+            let firstParam = 4;
+            let secondParam = 9;
+
+            let result = await deployedContract.sum(firstParam, secondParam);
+		    assert.equal(result, firstParam + secondParam, "Result is not expected one.");
+        });
+
+        it("Should execute function that accept 'address' as parameter successfully", async () => {
+
+            let result = await deployedContract.am_i_caller(ownerKeyPair.publicKey);
+            assert.equal(result, true, "Passed public key is not an owner!");
+            
+            result = await deployedContract.am_i_caller(notOwnerKeyPair.publicKey);
+		    assert.equal(result, false, "Passed public key is an owner!");
+        });
+
+        it("Should execute function that accept 'bool' as parameter successfully", async () => {
+            let param = true;
+
+            let result = await deployedContract.show_my_param(param);
+            assert.equal(result, param, "Returned value is not as the passed one, should be true");
+            
+            result = await deployedContract.show_my_param(!param);
+		    assert.equal(result, !param, "Returned value is not as the passed one, should be false");
+        });
+
+        it("Should execute function without parameter successfully", async () => {
+            let result = await deployedContract.get_caller();
+            assert.equal(result, ownerKeyPair.publicKey, "Returned value is not as the passed one, should be true");
+        });
+
+        it("Should execute function that accept AND 'amount/aettos' as second parameter", async () => {
+            let parameter = "Aleks";
+            
+            await assert.isFulfilled(deployedContract.sayHello(parameter, { value: 169}), "Function does not executed successfully!")
+        });
+
+        it("Should execute default [call] function.", async () => {
+            let param = `("I'm a super hero!")`;
+            let result = await deployedContract.call('sayHello', { "args": param });
+            let value = (await result.decode('string')).value;
+
+            assert.equal(value, `Hello ${param.replace(/[\(\)\")]+/g, '')}`, "Result is incorrect!");
+        });
+
+        it("Should execute default [call] function with passed amount/aettos.", async () => {
+            let param = `("I'm a super hero!")`;
+            let result = await deployedContract.call('sayHello', { "args": param, amount: 101 });
+		    let value = (await result.decode('string')).value;
+
+            assert.equal(value, `Hello ${param.replace(/[\(\)\")]+/g, '')}`, "Result is incorrect!");
+        });
+    });
+
+    describe("Test [from] functionality", async () => {
```

Comment:
here there should be a tests that can actually check that the function is called from given user